dask_ml/model_selection/_incremental.py

```diff
@@ -260,32 +288,40 @@ def get_futures(partial_fit_calls):
         _scores = {}
         _specs = {}
 
+        model_scores = {ident: info[ident][-1]["score"] for ident in instructions}
+        _sched_info = client._scheduler_identity
```

Comment:
Why is this needed? I'd prefer to avoid private APIs if possible. I believe you can use `client.scheduler_info()`.

Reply:
Should I test for asynchronous/synchronous client? It looks like the client returned by `gen_cluster` is asynchronous, so I'm not too worried.

Reply:
I get this error warning if I don't use the private attribute:

```
/Users/scott/anaconda3/envs/dask-ml-docs/lib/python3.6/site-packages/distributed/client.py:3479: RuntimeWarning: coroutine 'Client._update_scheduler_info' was never awaited
  self.sync(self._update_scheduler_info)

```

Reply:
More specifically, I get this error message when building the docs on TravisCI (and I believe on my own machine too).

``` shell
$ make html
...
>>>-------------------------------------------------------------------------

Warning in /home/travis/build/dask/dask-ml/docs/source/hyper-parameter-search.rst at block ending on line 460

Specify :okwarning: as an option in the ipython:: block to suppress this message

----------------------------------------------------------------------------

/home/travis/miniconda/envs/dask-ml/lib/python3.6/site-packages/distributed/client.py:3487: RuntimeWarning: coroutine 'Client._update_scheduler_info' was never awaited

  self.sync(self._update_scheduler_info)

<<<-------------------------------------------------------------------------
...
```